[quant][graphmode][fx] Add support for add

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43377 [quant][graphmode][fx] Add support for instance_norm
* #43376 [quant][graphmode][fx] Add support for layer_norm
* #43375 [quant][graphmode][fx] Add support for elu
* #43374 [quant][graphmode][fx] Add support for hardswish
* #43373 [quant][graphmode][fx] Add support for mul and mul relu
* #43372 [quant][grapphmode][fx][test][refactor] Refactor quantized add test
* #43335 [quant][graphmode][fx] Add support for batchnorm relu
* #43334 [quant][graphmode][fx] Add support for batchnorm
* #43333 [quant][graphmode][fx] Add support for cat
* #43332 [quant][graphmode][fx] Add support for add relu
* **#43331 [quant][graphmode][fx] Add support for add**
* #43287 [quant][graphmode][fx] Add support for conv module + relu

Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243561](https://our.internmc.facebook.com/intern/diff/D23243561)

jerryzh168 committed:
[quant][graphmode][fx] Add support for add

Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for add"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for add"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243561](https://our.internmc.facebook.com/intern/diff/D23243561)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for add"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243561](https://our.internmc.facebook.com/intern/diff/D23243561)

[ghstack-poisoned]

raghuramank100 commented on test/quantization/test_quantize_fx.py:
```diff
@@ -206,3 +207,109 @@ def forward(self, x):
+                x += y
+                return x
+
+        # TODO: decide whether we want to quantize or not
```
Why is this case not quantized?

jerryzh168 committed:
Update on "[quant][graphmode][fx] Add support for add"


Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23243561](https://our.internmc.facebook.com/intern/diff/D23243561)

[ghstack-poisoned]
